	For TL we prefer not to have files that differ just by the date.

diff -ur -x Makefile.in -x autom4te.cache xindy.orig/make-rules/inputenc/Makefile.am xindy/make-rules/inputenc/Makefile.am
--- xindy.orig/make-rules/inputenc/Makefile.am	2009-12-03 01:31:28.000000000 +0100
+++ xindy/make-rules/inputenc/Makefile.am	2009-12-07 23:42:02.000000000 +0100
@@ -41,7 +41,11 @@
 	latin3.xdy
 
 latin.xdy : latin1.xdy latin2.xdy latin3.xdy
+if TEXLIVE_BUILD
+	echo ';; Generated for TeX Live from $+' >latin.xdy
+else !TEXLIVE_BUILD
 	echo ';; Generated at' `date` 'from latin?.xdy' >latin.xdy
+endif !TEXLIVE_BUILD
 	echo >>latin.xdy
 	sort -u latin?.xdy | grep -v '^;' >>latin.xdy
 
